manifests/0000_90_ingress-operator_03_prometheusrules.yaml

```diff
@@ -53,3 +53,24 @@ spec:
           message: |
             The {{ $labels.namespace }}/{{ $labels.name }} ingresscontroller is
             unavailable: {{ $labels.reason }}.
+    - name: openshift-ingress-to-route-controller.rules
+      rules:
+        - alert: IngressWithoutClassName
+          expr: openshift_ingress_to_route_controller_ingress_without_class_name == 1
+          for: 1d
+          labels:
+            severity: warning
+          annotations:
+            summary: Ingress without IngressClassName
```

Comment:
Since the alert timing is unusually long, it might be worth it to mention the duration in the summary.
```suggestion
            summary: Ingress without IngressClassName for 1 day
```

Reply:
Applied.